src/Apps/Artwork/Components/OtherWorks/index.tsx

```diff
@@ -123,22 +118,14 @@ export const OtherWorks = track()(
           </Join>
         )}
 
-        {!(
-          context &&
-          context.__typename === "ArtworkContextAuction" &&
-          !(sale && sale.is_closed)
-        ) && (
-          <>
-            <Spacer mt={6} />
-
-            <RelatedWorksArtworkGridRefetchContainer artwork={artwork} />
-          </>
-        )}
-
+        {/*
+        TODO: Appears to never be true? Maybe this was supposed to match
+        context.__typename === "Sale"
+        or contextGrids[i] === "ArtworkContextAuction"
+        */}
```

Comment:
Little breadcrumb for the future — this was unrelated to my refactor, but in studying this code I began to think that the `OtherAuctionsQueryRenderer` below never actually gets rendered? 🤷🏽